pyhelm/chartbuilder.py

```diff
@@ -236,6 +233,15 @@ def get_helm_chart(self):
         self._helm_chart = helm_chart
         return helm_chart
 
+    def read_file(self, path):
+        '''
+        Open the file provided in `path` and strip any non-UTF8 characters.
+        Return back the cleaned content
+        '''
+        content = codecs.open(path, encoding='utf-8', errors='ignore').read()
```

Comment:
You need to close the file descriptor. You can wrap the ```read``` call with a ```with``` block doing the ```codes.open``` call

Reply:
Thanks, I missed that. Added in the latest commit